unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/WorldStateUtils.cs

```diff
@@ -0,0 +1,88 @@
+﻿using DCL.Controllers;
+using DCL.Helpers;
+using UnityEngine;
+
+namespace DCL
+{
+    public static class WorldStateUtils
+    {
+        public static bool IsGlobalScene(string sceneId)
+        {
+            var worldState = Environment.i.world.state;
+
+            if (worldState.TryGetScene(sceneId, out IParcelScene scene))
+            {
+                return scene is GlobalScene;
+            }
+
+            return false;
+        }
+
+        public static string TryToGetSceneCoordsID(string id)
+        {
+            var worldState = Environment.i.world.state;
+
+            if (worldState.loadedScenes.ContainsKey(id))
+                return worldState.loadedScenes[id].sceneData.basePosition.ToString();
+
+            return id;
+        }
+
+        public static Vector3 ConvertUnityToScenePosition(Vector3 pos, IParcelScene scene = null)
+        {
+            if (scene == null)
+            {
+                var worldState = Environment.i.world.state;
+
+                string sceneId = worldState.currentSceneId;
+
+                if (!string.IsNullOrEmpty(sceneId) && worldState.loadedScenes.ContainsKey(sceneId))
+                    scene = worldState.loadedScenes[worldState.currentSceneId];
```

Comment:
Can we use `TryGetValue` here instead of `ContainsKey`? also replace `worldState.currentSceneId` for `sceneId` inside de if statement?

Reply:
Moved this logic to another method to apply DRY.